CAIPs/caip-25.md

```diff
@@ -65,7 +95,7 @@ Example:
   "jsonrpc": "2.0",
   "method": "provider_authorization",
   "params": {
-    "requiredNamespaces": {
+    "requiredScopes": {
       "eip155": {
         "chains": ["eip155:1", "eip155:137"],
         "methods": ["eth_sendTransaction", "eth_signTransaction", "eth_sign", "get_balance", "personal_sign"],
```

Comment:
As discussed in today's editorial call: maybe using method names is not completely covering it - there is different behavior in methods between different clients (@timdaub surfaced this) 
My idea to attack it would be to create a registry (like we have namespaces) of possible methods - maybe defining them like here: https://github.com/ethereum/execution-apis/blob/main/src/eth/block.yaml - so we could cover e.g. the case of having one method_name with different parameters